stories/features/newTab/components/text/index.tsx

```diff
@@ -0,0 +1,81 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import styled, { css } from 'styled-components'
+import theme from '../../../../../src/theme/brave-default'
```

Comment:
This should not use the brave-default theme explicitly. Every component inside the `<ThemeWrapper` will automatically have `props.theme.x`. With this, we won't be able to change the theme.